internal/monitors/telegraf/monitors/procstat/procstat.go

```diff
@@ -83,6 +92,11 @@ func (m *Monitor) Configure(conf *Config) (err error) {
 	// create the emitter
 	em := baseemitter.NewEmitter(m.Output, logger)
 
+	// use the agent's configured host sys to get cgroup information
+	if conf.CGroup != "" {
```

Comment:
Shouldn't this be the opposite check for `CGroup` being blank?  It looks like you are setting a default here.

Reply:
If the cgroup name is specified, we explicitly build it with Host Sys.  This is to address behavior that's hard coded in the underlying telegraf plugin.  https://github.com/influxdata/telegraf/blob/master/plugins/inputs/procstat/procstat.go#L388

Reply:
ok, shouldn't the value that gets reassigned to `conf.CGroup` somehow incorporate the original `conf.CGroup` value?  Otherwise this is just saying if you set the CGRoup option to something it will always be overridden by this hardcoded value.

Reply:
yes, yes it should.  just fixed it.